install.md

```diff
@@ -0,0 +1,108 @@
+<!-- vim-markdown-toc GFM -->
+
+* [Nmstate Installation Guide](#nmstate-installation-guide)
+    * [RPM based](#rpm-based)
+        * [Stable release](#stable-release)
+        * [Developer Branch](#developer-branch)
+    * [PyPI/pip](#pypipip)
+        * [Stable Release](#stable-release-1)
+        * [Developer Branch](#developer-branch-1)
+    * [setup.py](#setuppy)
+        * [Stable Release](#stable-release-2)
+        * [Developer Branch](#developer-branch-2)
+
+<!-- vim-markdown-toc -->
+
+# Nmstate Installation Guide
+
+## RPM based
+
+### Stable release
+
+The nmstate is in stable repo of Fedora 28+ and EPEL7-testing repo, you may
```

Comment:
IMHO it would be better to write

`Nmstate is in Fedora and EPEL 7 testing, you may install it using the commands below:`

- I would avoid the version for Fedora, it will just look strange in a few years and there is no stable repo for Rawhide